src/core/manager.c

```diff
@@ -2699,7 +2676,7 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                         if (strv_extend(&m->deserialized_subscribed, l+11) < 0)
                                 log_oom();
 
-                } else
+                } else if (!startswith(l, "kdbus-fd=")) /* ignore this one */
```

Comment:
On Fri, Sep 09, 2016 at 09:57:13AM -0700, Evgeny Vereshchagin wrote:
> > @@ -2699,7 +2676,7 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
> >                          if (strv_extend(&m->deserialized_subscribed, l+11) < 0)
> >                                  log_oom();
> >  
> > -                } else
> > +                } else if (!startswith(l, "kdbus-fd=")) /* ignore this one */
> 
> Shouldn't we call `safe_close(fdset_remove(fds, kdbus_fd))` here?

We call safe_close() when cleaning up the fdset anyway, so I don't think
there's any need to call it explicitly.

Zbyszek